src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const total = 40 * days;
+  const withMinDiscount = total - 20;
+  const withMaxDiscount = total - 50;
+
+  if (days < 3) {
+    return total;
+  }
+
+  if (days >= 3 && days < 7) {
+    return withMinDiscount;
+  }
+
+  if (days >= 7) {
+    return withMaxDiscount;
+  }
```

Comment:
Functions should have at least 1 consistent return, and also you cover all the case before, so this if block is redundant
```suggestion
    return withMaxDiscount;
```